src/main/java/gregapi/block/multitileentity/MultiTileEntityItemInternal.java

```diff
@@ -37,14 +37,7 @@
 import gregapi.data.LH;
 import gregapi.data.MT;
 import gregapi.data.TD;
-import gregapi.item.CreativeTab;
-import gregapi.item.IItemColorableRGB;
-import gregapi.item.IItemEnergy;
-import gregapi.item.IItemGT;
-import gregapi.item.IItemNoGTOverride;
-import gregapi.item.IItemReactorRod;
-import gregapi.item.IItemRottable;
-import gregapi.item.IItemUpdatable;
+import gregapi.item.*;
```

Comment:
Is that a good idea?

Reply:
Shouldn't be breaking anything but I reverted it regardless